docs/reference/argocd.md

```diff
@@ -1024,6 +1025,30 @@ Name | Default | Description
 Enabled | false | Toggle Autoscaling support globally for the Argo CD server component.
 HPA | [Object] | HorizontalPodAutoscaler options for the Argo CD Server component.
 
+### Server Command Arguments
+
+Allows a user to pass arguments to Argo CD Server command.
+
+Name | Default | Description
+--- | --- | ---
+CommandArgs | [Empty] | List of arguments that will be added to Argo CD server command.
```

Comment:
Should make clear that it will append to the list of parameters already set by the Operator, and the behavior on override behavior.